src/components/ReleaseMonitor/__data__/mock-release-data.ts

```diff
@@ -0,0 +1,103 @@
+import { K8sResourceCommon } from '~/types/k8s';
+import { ReleaseKind, ReleaseCondition } from '~/types/release';
+
+export const mockReleases: ReleaseKind[] = [
```

Comment:
It seems the three releases are quite similar, how about we mock one then enjoy it to generate the later ones. Then the code would be shorter and cleaner. 
Another tips, we recommend that just leave required fields here .